activerecord/lib/active_record/connection_adapters/postgresql/oid/cidr.rb

```diff
@@ -28,6 +28,10 @@ def serialize(value)
             end
           end
 
```

Comment:
```suggestion
TODO: Remove when IPAddr#== compares IPAddr#prefix
```

Reply:
Added todo comment to remove method and issues url.